### DISCUSSION TITLE
Entanglefi omnichain partnerhip

### DISCUSSION THREAD

**Discussion by Brywall91111** [akash-network/community#723] [Governance Proposals]
Dear Community

Just thought I’d drop a quick note about our Universal Data Feeds (UDF)  and omnichain integrations– it could be a great fit for Akash

UDF’s all about real-time, accurate data, which could help with everything from risk assessment to adjusting interest rates on the fly. 

Plus, it’s designed to work across multiple chains, which might line up nicely with your multichain plans as we are partnered with 16+ chains.

Or We can also build anything you need omnichain to onboard new users from  all our 16 chains to Akash in a partnership.
It Would take us a few weeks to setup.

We can provide everything Chainlink is providing and better.

If you’re interested in chatting more, let me know – happy to dive into how it could support what you’re building!

UDF Docs: https://docs.entangle.fi/universal-data-feeds-guides/how-to-integrate

Live Feeds: https://datafeeds.entangle.fi/

Best wishes and Regards,

// @entanglefi $NGL

**Comment by ghost** [comment]
Thanks for posting 

Could you list more specific benefits for Akash network? 

What do you need from the Akash team to make this happen? 

What's the timeframe? 

And how much will this cost to the Akash community?
